content/en/cloud_cost_management/google_cloud.md

```diff
@@ -0,0 +1,124 @@
+
+## Setup
+
+### Enable Detailed Usage cost billing export
+<div class="alert alert-info"> <a href="https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables/detailed-usage">Detailed Usage Cost</a> data includes resource-level cost data. </div>
+
+1. Navigate to [Billing Export][1] under Google Cloud console *Billing*
+2. Enable [*Detailed Usage cost*][2] export, selecting or creating a project and a BigQuery dataset
+3. Document the billing account id for the billing account where the export was configured as well as export project id and dataset name
+
+### Create or select a Google Cloud Storage bucket
+You can use an existing GCS bucket or create a new one. Data is extracted regularly from your Detailed Usage Cost BigQuery dataset to a `datadog_cloud_cost_detailed_usage_export` prefix in the selected bucket.
+
+**Note:** The bucket [must be co-located][9] with the BigQuery export dataset.
+
+### Configure the Google Cloud Platform integration
+Navigate to [Setup & Configuration][3] and select an integrated Google Cloud Platform service account from the menu. If you do not see your desired Service Account in the list, go to [Google Cloud Platform integration][4] to add it. Follow the instructions on the integration page to properly configure your Service Account being integrated.
+
+<div class="alert alert-warning"> LEGACY project integrations are deprecated and not supported </div>
+
+Datadog Google Cloud Platform Integrations monitor the entire project when a related service account is integrated. Using a previously integrated project prevents monitoring resources in a new project. If your billing is associated with a non-integrated project, those resources will start to be monitored.
+
+**Note:** You can limit Metrics Collection per integration - this limits automatically monitoring resources in the project - but does not exclude data from Cloud Cost processing. </div>
+
+### Provide Datadog integration service account necessary permissions
+The APIs and permissions below enable Datadog to access your Detailed Usage billing export data and extract it in a useful format. This dumps data from BigQuery, where Google Cloud exports it, to your specified GCS bucket through a scheduled BigQuery query. The data is exported in a useful manner in your cloud storage bucket where it can then be processed by Datadog.
+
+#### Enable necessary Google Service APIs
+[Enable BigQuery and BigQuery Data Transfer Service APIs][5] if not already enabled
+
+#### Configure required bucket access
+[Add the service account as a principal on the GCS bucket resource][6] and assign a role with the following permissions:
+* `storage.buckets.get`
+* `storage.objects.create`
+* `storage.objects.delete`
+* `storage.objects.get`
+* `storage.objects.list`
+
+**Note:** This can be a custom role, or you can use existing Google Cloud roles `roles/storage.legacyObjectReader` and `roles/storage.legacyBucketWriter`
```

Comment:
```suggestion
**Note:** This can be a custom role, or you can use the existing Google Cloud roles `roles/storage.legacyObjectReader` and `roles/storage.legacyBucketWriter`.
```